packages/flutter_tools/lib/src/build_system/targets/native_assets.dart

```diff
@@ -64,17 +64,15 @@ class NativeAssets extends Target {
       }
       final TargetPlatform targetPlatform = getTargetPlatformForName(targetPlatformEnvironment);
       final Uri projectUri = environment.projectDir.uri;
-      final File packagesFile = fileSystem
-          .directory(projectUri)
-          .childDirectory('.dart_tool')
-          .childFile('package_config.json');
+
       final PackageConfig packageConfig = await loadPackageConfigWithLogging(
-        packagesFile,
+        environment.fileSystem.file(environment.packageConfigPath),
```

Comment:
nit looks like `fileSystem` is already a local variable here that's equivalent to `environment.fileSystem`

Reply:
Thanks!